docs/development/testguide.rst

```diff
@@ -27,17 +27,18 @@ dependency itself, in addition to several ``pytest`` plugins that are used by
 Astropy, and will also be of general use to other packages.
 
 Since the testing dependencies are not actually required to install or use
-Astropy, they are not included in ``install_requires`` in ``setup.cfg``.
-Instead, they are listed in an ``extras_require`` section called ``test`` in
-``setup.cfg``. Developers who want to run the test suite will need to either
+Astropy, they are not in the ``pyproject.toml`` included under the ``[project]``
+section in ``dependencies``.  Instead, they are listed under the
+``[project.optional-dependences]`` section called ``test`` in the ``pyproject.toml``
+file. Developers who want to run the test suite will need to either
 install pytest-astropy directly::
 
     pip install pytest-astropy
 
 or install the core package in 'editable' mode specifying the ``[test]``
 option::
 
-    pip install -e .[test]
+    pip install -e ".[test]"
```

Comment:
Are the quotes really necessary here?

Reply:
They are on Mac (because it uses zsh)

Reply:
@astrofrog is correct, that will break on `zsh` without the `"`, while the `"` does not change anything on other shells.